apps/frontend/static_src/scss/main.scss

```diff
@@ -1,29 +1,32 @@
 // Configuration
-@import "~bootstrap/scss/functions";
+@import "./vendor/normalize";
 
 //custom font
 @import "./kumbh-sans.scss";
 
 //custom variables
 @import "./variables.scss";
+@import "./functions.scss";
 @import "./mixins.scss";
 
+@import "~bootstrap/scss/functions";
+
 @import "~bootstrap/scss/variables";
 @import "~bootstrap/scss/mixins";
 @import "~bootstrap/scss/utilities";
 
-// Layout & components
-@import "~bootstrap/scss/root";
-@import "~bootstrap/scss/reboot";
-@import "~bootstrap/scss/type";
+// // Layout & components
+// @import "~bootstrap/scss/root";
+// @import "~bootstrap/scss/reboot";
+// @import "~bootstrap/scss/type";
```

Comment:
This worries me a bit. I’d have assumed lots / all of the Bootstrap components we are using assume they’re built upon Bootstrap’s `root` and `reboot`. Is there any way we could keep those from Bootstrap instead of replacing with normalize?

If we do want to keep normalize – then I think it’d be nice to use the latest version from https://necolas.github.io/normalize.css/ rather than an older one.

Reply:
yea I agree, I'm hoping to eventually remove all of the bootstrap elements, or at least all of the styling (just leaving the functionality for modals/dropdowns etc that are already implemented). I'll update the normalize for the time being.